client/landing/jetpack-cloud/components/log-item/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import CardHeading from 'components/card-heading';
+
+import './style.scss';
+
+interface Props {
+	children?: ReactNode;
+	header: string;
+	subheader: string;
+	highlight?: string;
+	tag?: string;
+}
+
+class LogItem extends React.PureComponent< Props > {
+	static defaultProps = {
+		highlight: 'info',
+		tag: 'new',
+	};
+
+	renderHeader() {
+		const { header, subheader, tag } = this.props;
+
+		return (
+			<div className="log-item__header-wrapper">
+				<span className="log-item__tag">{ tag }</span>
+				<CardHeading tagName="h2" size={ 18 }>
+					{ header }
+				</CardHeading>
+				<p className="log-item__subheader">{ subheader }</p>
+			</div>
+		);
+	}
+
+	render() {
+		const { highlight, children } = this.props;
+		return (
+			<FoldableCard header={ this.renderHeader() } className="log-item" highlight={ highlight }>
```

Comment:
Given that this component will be a strictly Jetpack Cloud component for now, should we be prefixing the CSS classes of those components with `jetpack-cloud__` or something?

Reply:
I attempted to, but it threw a lot of warnings. It seems that [the ESlint rule](https://github.com/Automattic/wp-calypso/blob/e888a84274179cebd44fdebf391b99cec8500cbe/packages/eslint-plugin-wpcalypso/docs/rules/jsx-classname-namespace.md) doesn't allow for this.

Reply:
Yeah, because it's in a subdirectory. Alright. let's keep it as-is for now.